### PR TITLE
PB-88: Added import maps E2E tests

### PR DESCRIPTION
**Pull request by ltshb**
TODO in separate PR:
- [x] WMS import edge case at startup see #618 
- [x] WMTS import see #618 
- [x] WMTS import edge case at startup see #618 

[Test link](https://sys-map.dev.bgdi.ch/preview/feat-pb-88-import-e2e/index.html)

### CODE REVIEW

**Review by ismailsunni — APPROVED** [review]
LGTM, just one minor comment

File: src/modules/menu/components/share/MenuShareEmbed.vue [hunk 1]
```diff
@@ -2,7 +2,6 @@
     <div class="menu-share-embed">
         <a
             class="embed-btn d-flex align-items-center text-decoration-none ps-3 p-2"
-            :class="{ 'text-primary': isSelected, 'text-black': !isSelected }"
```
[the remark this answers is not in the document]
Comment (ltshb) [reply src/modules/menu/components/share/MenuShareEmbed.vue:5]: This line was introduced in a previous pr #614 (see commit message), it was a copy/paste mistake and in this component there is no `isSelected` ref and Vue was generating a warning
